runtime/port/unix/j9sysinfo.c

```diff
@@ -1668,6 +1668,16 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 			omrcpu_get_cache_line_size(&result);
 #endif
 		}
+#elif defined(LINUX) && defined(J9AARCH64)
+	if ((query->cmd == J9PORT_CACHEINFO_QUERY_LINESIZE)
+	    && (query->cacheType == J9PORT_CACHEINFO_DCACHE)
+	    && (query->level == 1)) {
```

Comment:
Why is the formatting here different than the PR for `master`?   Please also port 078394b778c0ce2a022bc596b74523a49a20c183 over so this code exactly matches the `master` version

Reply:
Just a mistake copying my older local version to this commit.
Corrected.